src/resources/extensions/quarto/confluence/overrides.lua

```diff
@@ -115,18 +115,15 @@ function CaptionedImageConfluence(source, title, caption, attr)
 end
 
 function LinkConfluence(source, target, title, attr)
-  local LINK_ATTACHMENT_SNIPPET = [[<ac:link><ri:attachment ri:filename="{source}"/><ac:plain-text-link-body><![CDATA[{target}{doubleBraket}></ac:plain-text-link-body></ac:link>]]
+  -- For some reason, rendering converts spaces to a double line-break
```

Comment:
I don't know why the Pandoc rendering is adding these line breaks.  @dragonstyle Do you have any insights as to why this might be?